crates/bdk/Cargo.toml

```
@@ -15,8 +15,9 @@ rust-version = "1.57"
 [dependencies]
 log = "=0.4.18"
 rand = "^0.8"
-miniscript = { version = "9", features = ["serde"], default-features = false }
-bitcoin = { version = "0.29", features = ["serde", "base64", "rand"], default-features = false }
+miniscript = { version = "10", features = ["serde"], default-features = false }
+bitcoin = { version = "0.30", features = ["serde", "base64", "rand"], default-features = false }
+secp256k1 = { version = "0.27", features = [ "rand-std" ] }
```

```suggestion
bitcoin = { version = "0.30", features = ["serde", "base64", "rand-std"], default-features = false }
```
I guess the secp dependency was just to enable "rand-std"? `rust-bitcoin` has "rand-std" that can do that for you.